ivy/functional/backends/torch/random.py

```diff
@@ -37,14 +42,14 @@ def random_normal(
     shape: Optional[Union[ivy.NativeShape, Sequence[int]]] = None,
     *,
     device: torch.device,
+    dtype: torch.dtype,
     out: Optional[torch.Tensor] = None,
 ) -> torch.Tensor:
     if isinstance(mean, numbers.Number) and isinstance(std, numbers.Number):
         ret = torch.normal(mean, std, ivy.default(shape, ()), out=out)
+    if isinstance(mean, ivy.Array) and isinstance(std, ivy.Array):
+        ret = torch.normal(float(mean), float(std), ivy.default(shape, ()), out=out)
     else:
-        assert shape is None, (
```

Comment:
I removed this assertion because it fails the test when mean and std are given as Tensors.
Could you explain why this assertion was included?